docs/versioning-policy.md

```diff
@@ -0,0 +1,98 @@
+You must **not** rely on these numbers to derive any information about breaking changes or ease of updating.
+**This is not SemVer!**
+
+However, we do care a lot about providing you with all relevant information about breaking changes!
+Every release mentions all potentially breaking changes in its changelog.
+Additionally, there is a release overview with breaking changes [here](./releases.md).
+
+<br>
+<br>
+
+---
+
+## Breaking and non-breaking changes
+
+Our definition of these terms for Tobira. Defines what we guarantee and what not.
+
+### Breaking changes/stability guarantees
+
+We consider the following changes as *breaking*, highlighting them in changelogs and tracking them in [the release overview](./releases.md).
+The short tags at the beginning are the table column headers in said release overview.
+
+- *config*: Changing the configuration file in a backwards-incompatible way.
+- *auth*: Changing the auth-system in a backwards-incompatible way.
+- *cli*: Changing the stabilized parts of the CLI in a backwards-incompatible way.
+- *meili*: Requiring a new version of MeiliSearch.
+- *pg*: Dropping support for a non-EOL version of PostgreSQL.
+- *tmod*: Requiring a newer version of the Tobira module in Opencast.
+- *ocversion*: Requiring a new Opencast version.
+- *ocreq*: Adding new Opencast requirements (e.g. having static-file authorization disabled).
+
+
+### Non-breaking changes
+
+All changes not listed in the previous section are considered *non-breaking* by us.
+We may release these changes at any time.
+And while most will likely be mentioned in the changelog somewhere, some may not.
+
+To name a few things explicitly (this is not a complete list!):
+
+- Changing the configuration file in a backwards-compatible way (e.g. adding new fields with default values, allowing more values for an existing field, ...).
+- Adding new non-required `x-tobira-*` headers to the auth system.
+- Dropping support for an [EOL-version of PostgreSQL](https://www.postgresql.org/support/versioning/) (e.g. PostgreSQL 10 after Nov 10, 2022)
+- Database schema changes: we always provide scripts to easily migrate the database.
+  Tobira automatically runs the appropriate scripts when starting, so you don't have to do anything.
+- Anything that requires a search index rebuild.
+  These rebuilds will be performed automatically after updating Tobira and should be done within a few seconds (for tens of thousands of events/series).
+- Any changes to the GraphQL API. **This API is internal and we will break it very regularly.**
+- Any changes to emitted HTML, DOM nodes, CSS, and JS.
+- Any user-facing changes, like adding features, removing features, changing features, change the design, ...
+
+
+<br>
+<br>
+
+---
+
+## Stabilized parts of Tobira
+
+See the section "Breaking and non-breaking changes" for context.
+
+### CLI (command line interface)
```

Comment:
This isn't really anything closely related to versioning policy. I would probably drop this whole section.

Reply:
At least some CLI parts (e.g. `tobira serve`) are certainly very important as we cannot just change them. Otherwise the service definitions and stuff will break for admins. At the same time, we have lots of ad-hoc and utility sub-commands in the CLI. So we have to at least treat some CLI changes as breaking, but treating all as breaking is also overkill. So I think it's a good idea to just make a very small part (thats needed for a standard deployment) "stable".

Reply:
If we ever change these commands, that will break people, so I'd have to agree that it's valid to mention it here. However, it looks a bit weird to have only the CLI listed under "stable parts." :thinking: Plus, we already mention above that CLI stability is something we will report on as part of the release process. :shrug: 

Again no hard opinion, though.

Reply:
I removed this section alltogether now. See new commit.